npm/README.md

```diff
@@ -36,10 +36,7 @@
 
 ### Introduction
 
-**[Infisical](https://infisical.com)** is the open source secret management platform that teams use to centralize their application configuration and secrets like API keys and database credentials as well as manage their internal PKI.
-
-We're on a mission to make security tooling more accessible to everyone, not just security teams, and that means redesigning the entire developer experience from ground up.
-
+The Infisical CLI is powerful command line tool that can be used to retrieve, modify, export and inject secrets into any process or application as environment variables. You can use it across various environments, whether it’s local development, CI/CD, staging, or production.
```

Comment:
same typo; missing a -> `is a powerful command...`

thoughts on something like:

"The Infisical CLI streamlines secret management by letting you retrieve, modify, and inject environment variables across your entire development pipeline. From local development to production deployments, it integrates seamlessly with your applications and workflow."

Reply:
Let's stick with what we have right now (with the typo fixed). I've copied this directly from our CLI documentation actually. I will update the typo in our docs as well